pkgs/development/gnuradio-modules/osmosdr/default.nix

```diff
@@ -27,24 +29,33 @@ let
   version = {
     "3.7" = "0.1.5";
     "3.8" = "0.2.3";
+    "3.9" = "0.2.4";
+    "3.10" = "0.2.4";
   }.${gnuradio.versionAttr.major};
   src = fetchgit {
     url = "git://git.osmocom.org/gr-osmosdr";
     rev = "v${version}";
     sha256 = {
       "3.7" = "0bf9bnc1c3c4yqqqgmg3nhygj6rcfmyk6pybi27f7461d2cw1drv";
       "3.8" = "sha256-ZfI8MshhZOdJ1U5FlnZKXsg2Rsvb6oKg943ZVYd/IWo=";
+      "3.9" = "sha256-d0hbiJ44lEu8V4XX7JpZVSTQwwykwKPUfiqetRBI6uI=";
+      "3.10" = "sha256-d0hbiJ44lEu8V4XX7JpZVSTQwwykwKPUfiqetRBI6uI=";
     }.${gnuradio.versionAttr.major};
   };
 in mkDerivation {
   pname = "gr-osmosdr";
   inherit version src;
-  disabledForGRafter = "3.9";
+  disabledForGRafter = "3.11";
 
   outputs = [ "out" "dev" ];
 
   buildInputs = [
+  ] ++ (if (lib.versionAtLeast gnuradio.versionAttr.major "3.10") then [
+    libsndfile
```

Comment:
The build is currently failing for gnuradio 3.9 because libsndfile is only added to gnuradio 3.10. If added to 3.9 it compiles fine. 
Also, perhaps libsndfile should be propagated from gnuradio?

Reply:
Thanks for noticing this issue, I missed @jiegec's build failure reports. Regarding `log4cpp` and `spdlog` it's sort of justified to propagate it, but I still tend to not do it. As for `libsndfile`, I'm rather sure it's not needed on almost all `gnuradioPackages`. I did change that conditional to depend upon whether gnuradio has the `gr-blocks` feature compiled into it.